ironic-deployment/ironic/ironic.yaml

```diff
@@ -96,6 +159,20 @@ spec:
         - name: ironic-inspector
           image: quay.io/metal3-io/ironic
           imagePullPolicy: Always
+          readinessProbe:
+            exec:
+              command: ["sh", "-c", "curl -sSf http://127.0.0.1:5050 || curl -sSf -k https://127.0.0.1:5050"]
+            failureThreshold: 3
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          livenessProbe:
+            failureThreshold: 3
+            exec:
+              command: ["sh", "-c", "curl -sSf http://127.0.0.1:5050 || curl -sSf -k https://127.0.0.1:5050"]
```

Comment:
I think the port should be 5059 based on https://github.com/metal3-io/baremetal-operator/pull/975#discussion_r708355026 ?

Reply:
you mean 5049 ?
I think we discussed with @namnx228 , the port 5050 always works. The 5049 is dependent on the TLS setup and it may not work. @namnx228 , any comment ?

Reply:
yes sorry, 5049

Reply:
Yes, the port 5050 should always work. In details, we have two cases:
- Not using TLS: in this case, the port of ironic-inspector is 5050
- Using TLS: The port of the reverse proxy is 5050, and of the ironic-inspector is 5049. However, every requests sent to 5050 should be forwarded to 5049, and if ironic-inspector is down, replies from the reverse proxy should not have the status code 200.

Reply:
Thanks @namnx228.
@fmuyassarov I am of the opinion that asserting on 5050 is sufficient.

Reply:
okay sounds good. Thanks for explanation @namnx228 !